test/chartmogul/plan-group.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const ChartMogul = require('../../lib/chartmogul');
+const config = new ChartMogul.Config('token', 'secret');
+const expect = require('chai').expect;
+const nock = require('nock');
+const PlanGroup = ChartMogul.PlanGroup;
+
+describe('PlanGroup', () => {
+  it('should create a new plan group', () => {
+    /* eslint-disable camelcase */
+    const postBody = {
+      name: 'Bronze Plan Group',
+      plans: ['pl_eed05d54-75b4-431b-adb2-eb6b9e543206', 'pl_ab225d54-7ab4-421b-cdb2-eb6b9e553462']
+    };
+    /* eslint-enable camelcase */
+
+    nock(config.API_BASE)
+      .post('/v1/plan_groups', postBody)
+      .reply(200, {
+        /* eslint-disable camelcase */
+        name: 'Bronze Plan Group',
+        uuid: 'plg_eed05d54-75b4-431b-adb2-eb6b9e543206',
+        plans: [
```

Comment:
[API design] how many plans can be in the array? If there's a limit, how can it be paged? I would expect similar response to `GET`.

Reply:
Right, need to figure that one out 🍡